### PR TITLE
[docs] Describe adding custom methods to selector

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs

### CODE REVIEW

File: docs/articles/documentation/test-api/selecting-page-elements/selectors.md [hunk 1]
```diff
@@ -728,6 +732,44 @@ test('Check Label HTML', async t => {
...
+`node`                  | Object   | The DOM node.
+`param1`, `param2`, ... | Any      | Method parameters.
+
+The `addCustomMethods` function also adds the specified methods to the [element state](#obtain-element-state) returned by the selector.
+
+**Example**
+
+```js
+const myTable = Selector('.my-table').addCustomMethods({
+    getCellText: (table, rowIndex, columnIndex) => {
```
Comment (churkin) [inline docs/articles/documentation/test-api/selecting-page-elements/selectors.md]:
```javascript 
getCellText: (table, rowIndex, columnIndex) => table.rows[rowIndex].cells[columnIndex].innerText
```

Comment (VasilyStrelyaev) [inline docs/articles/documentation/test-api/selecting-page-elements/selectors.md]: I don't like horizontal scroll bars below the code

Comment (churkin) [inline docs/articles/documentation/test-api/selecting-page-elements/selectors.md]: You can start it from new line

Comment (churkin) [inline docs/articles/documentation/test-api/selecting-page-elements/selectors.md]:
```javascript
getCellText: (table, rowIndex, columnIndex) => 
    table.rows[rowIndex].cells[columnIndex].innerText
```

I like it more

File: docs/articles/documentation/test-api/selecting-page-elements/selectors.md [hunk 2]
```diff
@@ -728,6 +732,44 @@ test('Check Label HTML', async t => {
...
+        return table.rows[rowIndex].cells[columnIndex].innerText;
+    }
+});
+
+await myTable.getCellText(2,5);
```
Comment (inikulin) [inline docs/articles/documentation/test-api/selecting-page-elements/selectors.md]: It would be nice usage of that in assertions

**Comment by VasilyStrelyaev** [comment]
FPR

File: docs/articles/documentation/test-api/selecting-page-elements/selectors.md [hunk 3]
```diff
@@ -38,7 +38,9 @@ This topic contains the following sections.
   * [Define Action Targets](#define-action-targets)
   * [Define Assertion Actual Value](#define-assertion-actual-value)
   * [Selector Timeout](#selector-timeout)
-* [Adding Custom Properties to Element State](#adding-custom-properties-to-element-state)
+* [Adding Custom Properties and Methods to the Element State](#adding-custom-properties-and-methods-to-the-element-state)
```
Comment (inikulin) [inline docs/articles/documentation/test-api/selecting-page-elements/selectors.md]: `Extending selectors` maybe?

Comment (VasilyStrelyaev) [inline docs/articles/documentation/test-api/selecting-page-elements/selectors.md]: maybe

**Pull request merged**